src/main/java/com/tasks/taskapi/controller/TaskController.java

```diff
@@ -23,6 +24,12 @@ public ResponseEntity<List<Task>> getAllTasks() {
        return ResponseEntity.status(HttpStatus.OK).body(tasks);
     }
 
+    @PostMapping("/post")
+    public ResponseEntity<Task> insertTask(Task taskToInsert) {
+        Task insertedTask = taskService.insertTask(taskToInsert);
```

Comment:
insertedTask and taskToInsert are very similar, might be more readable if you use more different variable names